### ISSUE TITLE
TODO list - several action in the list

### ISSUE THREAD

**Issue by dapgo** [enaon/ninebot-one-nRF52#3]
A new feature to store display/notes

**User case1: (Send and display)**

1.   User from a BLE app, send a Note to the smartwatch
    i.e. through a command like:
    _AT+PUSH=TODO,1,"todoaction1" or AT+PUSH=TODO1,"todoaction1"_
    Smartwatch software identifies that it is a TODO entry and stores it on slot x (1)

2.   A user, from Smartwatch Menus, click on "TODO"
     _Smartwatch screen displays "ToDo List" page and several lines (a TODO entry per line) 
      (i.e Entries from 1 to 5)_    

**User case2: (Moving through multiple pages)**

1.  A user, from Smartwatch Menus, click on "TODO"
    Smartwatch screen displays "ToDo List" page and several entries (i.e from 1 to 5)

2.  User can move to next "ToDo List" page, and display next entries 
    Smartwatch screen displays "ToDo List (2)" page and next entries (i.e from 6 to 10)
   
**User case3: (Overwrite/replace an existing TODO entry)**

1.  User from a BLE app, send a new TODO to the smartwatch to overwrite a slot (i.e. todo3)
    i.e AT+PUSH=TODO,3,"newtxt_to_replace_ToDo3"

**Comment by enaon** [comment]
this is now supported:

AT+PUSH=9,todo_name,todo_body,3"

the leading 9 indicates a todo, the last number is the todoslot. Leave the todo_body empty to delete the todo.

**Comment by dapgo** [comment]
At the moment the number for TODO is 8
AT+PUSH=8,headerTODO,bodyTODO,3

Can TODO list have a tickbox associated to each entry/line? 

- [ ]  TODO entry1(unticked)

- [x]  TODO entry3(ticked /completed)

**Comment by enaon** [comment]
yes but no need to send the tickbox I believe, we can handle that on the push face. I will add that, nice.
